scripts/protcon_analyisis/protcon_analyisis.yml

```diff
@@ -0,0 +1,72 @@
+script: protcon_analyisis.R
```

Comment:
There is a typo in the file name, this should be changed

Reply:
done

Reply:
yes!, sorry and all the calls in others files